### PR TITLE
Add alerts for when claims info is not available

### PR DESCRIPTION
**Pull request by jbalboni**
Closes [68](https://github.com/department-of-veterans-affairs/sunsets-team/issues/68).

When out of sync:
screen shot 2016-10-13 at 2 01 22 pm

When an error is returned from backend:
screen shot 2016-10-13 at 2 01 37 pm

### CODE REVIEW

**Comment by jbalboni** [comment]
@austin-martinez @cagedcrown

File: src/js/disability-benefits/components/DetailLayout.jsx [hunk 1]
```diff
@@ -1,47 +0,0 @@
-import React from 'react';
```
Comment (jbalboni) [inline src/js/disability-benefits/components/DetailLayout.jsx]: This was a dupe checked in on my last PR.

File: src/js/disability-benefits/reducers/claim-sync.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+export default function claimDetailReducer(state = initialState, action) {
+  switch (action.type) {
+    case SET_CLAIM_DETAIL:
+      return _.merge(state, {
+        synced: action.claim.attributes.successfulSync,
+        syncedDate: action.claim.attributes.updatedAt,
+        available: true
+      });
+    case SET_CLAIMS:
+      if (action.claims.length) {
```
Comment (austin-martinez) [inline src/js/disability-benefits/reducers/claim-sync.js]: What is this conditional for? What if the user just doesn't have any claims?

Comment (jbalboni) [inline src/js/disability-benefits/reducers/claim-sync.js]: If they don't have any claims, we don't want to show either of these messages. They'll get a message on the claims list page if their list is empty, which is handled on the list page.

**Comment by austin-martinez** [comment]
LGTM :100:

**Pull request merged**